### ISSUE TITLE
Error installing R package on Windows

### ISSUE THREAD

**Issue by PetoLau** [chlubba/catch22#2]
Hi,

I have problem to install your package from R...following errors occurred:

* installing *source* package 'catch22' ...
** libs
c:/Rtools/mingw_64/bin/gcc  -I"C:/PROGRA~1/R/R-35~1.1/include" -DNDEBUG -I../inst/include -I"C:/Users/PeterLaurinec/Documents/R/win-library/3.5/Rcpp/include"        -O2 -Wall  -std=gnu99 -mtune=generic -c CO_AutoCorr.c -o CO_AutoCorr.o
In file included from CO_AutoCorr.c:8:0:
CO_AutoCorr.c: In function 'CO_AutoCorr':
fft.h:16:43: error: '_Imaginary_I' undeclared (first use in this function)
 #define CMPLX(x, y) ((cplx)((double)(x) + _Imaginary_I * (double)(y)))
                                           ^
CO_AutoCorr.c:63:16: note: in expansion of macro 'CMPLX'
         F[i] = CMPLX(y[i] - m, 0.0);
                ^
fft.h:16:43: note: each undeclared identifier is reported only once for each function it appears in
 #define CMPLX(x, y) ((cplx)((double)(x) + _Imaginary_I * (double)(y)))
                                           ^
CO_AutoCorr.c:63:16: note: in expansion of macro 'CMPLX'
         F[i] = CMPLX(y[i] - m, 0.0);
                ^
CO_AutoCorr.c: In function 'co_autocorrs':
fft.h:16:43: error: '_Imaginary_I' undeclared (first use in this function)
 #define CMPLX(x, y) ((cplx)((double)(x) + _Imaginary_I * (double)(y)))
                                           ^
CO_AutoCorr.c:97:16: note: in expansion of macro 'CMPLX'
         F[i] = CMPLX(y[i] - m, 0.0);
                ^
CO_AutoCorr.c: In function 'CO_HistogramAMI_even_2_5':
CO_AutoCorr.c:347:5: error: variable-sized object may not be initialized
     double binEdges[numBins+1] = {0};
     ^
CO_AutoCorr.c:347:5: warning: excess elements in array initializer
CO_AutoCorr.c:347:5: warning: (near initialization for 'binEdges')
CO_AutoCorr.c:370:5: error: variable-sized object may not be initialized
     double binEdges12[(numBins + 1) * (numBins + 1)] = {0};
     ^
CO_AutoCorr.c:370:5: warning: excess elements in array initializer
CO_AutoCorr.c:370:5: warning: (near initialization for 'binEdges12')
CO_AutoCorr.c:414:5: error: variable-sized object may not be initialized
     double pi[numBins] = {0};
     ^
CO_AutoCorr.c:414:5: warning: excess elements in array initializer
CO_AutoCorr.c:414:5: warning: (near initialization for 'pi')
CO_AutoCorr.c:415:5: error: variable-sized object may not be initialized
     double pj[numBins] = {0};
     ^
CO_AutoCorr.c:415:5: warning: excess elements in array initializer
CO_AutoCorr.c:415:5: warning: (near initialization for 'pj')
make: *** [C:/PROGRA~1/R/R-35~1.1/etc/x64/Makeconf:208: CO_AutoCorr.o] Error 1
ERROR: compilation failed for package 'catch22'

Any idea?
Thx

**Comment by chlubba** [comment]
I will close this thread now. There have been some updates to the code, especially on the `_Imaginary_I` compile problem. If it still does not work for you, thanks for opening another issue.

Best,
Carl
